tests/test_run_state_async.py

```
@@ -0,0 +1,38 @@
+    manager = RunStateManager(runs_dir)
+
+    # Create dummy runs
+    for i in range(3):
+        run_id = f"run-{i}"
+        run_dir = runs_dir / run_id
+        run_dir.mkdir()
+
+        state = {
+            "run_id": run_id,
+            "status": "completed",
+            "created_at": datetime.now(timezone.utc).isoformat(),
```

Ruff (0.14.14)

[warning] Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)